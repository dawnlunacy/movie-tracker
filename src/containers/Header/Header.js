import React from 'react';
import { Link } from 'react-router-dom';
import { connect } from 'react-redux';
import logo from '../../images/MovieTracker_font_wave.png';
import Nav from '../Nav/Nav';

import PropTypes from 'prop-types';

export const Header = ({ getFavorites }) => {

  return(
    <header className="App-header">
      <Nav getFavorites={getFavorites}/>
      <Link className="App-img" to='/'>
      <img src={logo} alt="Logo" />
      </Link>
    </header>
  )

  }
//       <Nav getFavorites= { getFavorites }/>
//       <img src={logo} alt="Logo" className="App-img"/>
//     </header>
//   )
// }

Nav.propTypes = {
  getFavorites: PropTypes.func.isRequired
}

// onClick={browserHistory.goBack}