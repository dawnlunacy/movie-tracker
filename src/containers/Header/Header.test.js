import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  it('should match snapshot', () => {
    const getFavorites = jest.fn()
    const wrapper = shallow(<Header getFavorites={getFavorites}/>);
    
    expect(wrapper).toMatchSnapshot();
  });
});