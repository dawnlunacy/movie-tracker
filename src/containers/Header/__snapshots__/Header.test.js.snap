// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header
  className="App-header"
>
  <Connect(Nav)
    getFavorites={[MockFunction]}
  />
  <Link
    className="App-img"
    to="/"
  >
    <img
      alt="Logo"
      src="MovieTracker_font_wave.png"
    />
  </Link>
</header>
`;
